setup.py

```diff
@@ -30,6 +30,7 @@
         "click >= 7.0",
         "stringcase >= 1.2.0, < 2.0.0",
         "requests >= 2.21.0",
+        "fuzzywuzzy[speedup] >= 0.18.0",
```

Comment:
Does the `[speedup]` contain lots of C that will make ftm less likely to run on people's computers? We're already slightly fucked on that front because of `pyicu`, just to check...

Reply:
No. It ensures that `python-Levenshtein` is installed.